packages/web-frontend/src/components/MapWatermark.js

```diff
@@ -0,0 +1,38 @@
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: 65px 20px;
+`;
+
+export const MapWatermark = props => (
+  <StyledLink
+    href="http://mapbox.com/about/maps"
+    className="mapbox-wordmark"
+    target="_blank"
+    rel="noreferrer"
+    {...props}
+  >
+    Mapbox
+  </StyledLink>
```

Comment:
Similarly, perhaps we'd want to move this link into the expanded info list on mobile to prevent accidentally click it?